tests/sentry/web/frontend/test_auth_organization_login.py

```diff
@@ -85,7 +85,7 @@ def test_flow_as_anonymous(self):
 
         user = auth_identity.user
         assert user.email == "foo@example.com"
-        assert not user.has_usable_password()
+        assert not user.password
```

Comment:
While all that's needed is just to fix this test I think we should not use `has_usable_password`. Mostly confident in this, need to look at `createuser` and the sudo powers assumed thing.